src/components/Contacts/ContactDetails/ContactDetailsTab/People/Items/EditPersonModal/PersonShowMore/PersonShowMore.tsx

```diff
@@ -44,19 +64,30 @@ export const PersonShowMore: React.FC<PersonShowMoreProps> = ({ person }) => {
               </InputLabel>
               <Select
                 labelId="relationship-status-label"
-                value={person.maritalStatus}
+                value={maritalStatus}
+                onChange={handleChange('maritalStatus')}
                 fullWidth
               >
-                <MenuItem value="single">{t('Single')}</MenuItem>
+                <MenuItem value="Single">{t('Single')}</MenuItem>
+                <MenuItem value="Engaged">{t('Engaged')}</MenuItem>
+                <MenuItem value="Married">{t('Married')}</MenuItem>
+                <MenuItem value="Separated">{t('Separated')}</MenuItem>
+                <MenuItem value="Divorced">{t('Divorced')}</MenuItem>
+                <MenuItem value="Widowed">{t('Widowed')}</MenuItem>
```

Comment:
Any chance the API provides these as an enum? Guess we'd still need to translate them client side.

Reply:
No, the api just accepts any string. The client handles providing the specific expected string. Maybe it would be better as an enum?